js/ popup.js

```diff
@@ -0,0 +1,56 @@
+const bigPicture = document.querySelector('.big-picture');
+const bigPictureSrc = bigPicture.querySelector('img');
+const bigLikes = document.querySelector('.likes-count');
```

Comment:
```suggestion
const bigLikes = bigPicture.querySelector('.likes-count');
```

И ниже проверь где можно оптимизировать/сузить поиск элементов